src/RageSurface_Load_JPEG.cpp

```diff
@@ -27,10 +27,13 @@ struct my_jpeg_error_mgr
 {
   struct jpeg_error_mgr pub;    /* "public" fields */
 
+  uint8_t pad1[8];  // Explicitly add padding that's implicitly added by the compiler
```

Comment:
Hmm I feel like we shouldn't be manually adding this padding. Can we instead rearrange the variables such that they get packed better?